roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -28,11 +28,10 @@ cloud:
 discovery:
   type: kubernetes
   zen.ping.multicast.enabled: false
-  zen.minimum_master_nodes: {{es_min_masters}}
+  zen.minimum_master_nodes: {{es_node_quorum}}
```

Comment:
i think they should remain as env vars, especially since we use a single configmap for ES (and ops). Having this value in a variable would allow us to scale ops and non-ops differently and not cause one to go haywire waiting for a number of master nodes that may never become available